lib/topological_inventory/host_inventory_sync.rb

```diff
@@ -173,17 +174,22 @@ def default_messaging_opts
       }
     end
 
-    def build_topological_inventory_url(passed_url)
-      return passed_url if passed_url
-
+    def build_topological_inventory_url
       path = File.join("/", ENV["PATH_PREFIX"].to_s, ENV["APP_NAME"].to_s, TOPOLOGICAL_INVENTORY_API_VERSION)
 
-      require "uri"
       URI::HTTP.build(
         :host => ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_HOST"],
         :port => ENV["TOPOLOGICAL_INVENTORY_API_SERVICE_PORT"],
         :path => path
       ).to_s
     end
+
+    def build_host_inventory_url
+      path = File.join("/", ENV["PATH_PREFIX"].to_s, "inventory", "api", "v1")
```

Comment:
hm looking at this, the env vars should not be really here, we should use the class interface only